tests/provisioning/README.md

```diff
@@ -0,0 +1,13 @@
+# Tests for Stackdriver Sandbox Provisioning
+
+To run the integration test for the provisioning, follow the steps below.
+
+1. Install the dependencies
+```bash
+pip3 install -r requirements.txt
+```
+
+2. Run the tests
+```bash
+python3 provision_test.py
```

Comment:
This should be part of our CI @jiang95-dev

Reply:
Yes, you should follow the patter Nick uses here: https://github.com/GoogleCloudPlatform/stackdriver-sandbox/pull/335